DEV_README.md

```diff
@@ -0,0 +1,25 @@
+# pack3d
```

Comment:
Exactly the kind of file we need. standard name for this is CONTRIBUTING.md  (contributors doc)